### DISCUSSION TITLE
How to bulk update selected rows in a table ?

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#5666] [Help]
I was trying to have this functionality in the app , where a user can select the rows which would required a column in the selected row to get updated with a values where they can choose from a dropdown filed and save .
How to do this functionality?
<img width="960" alt="save" src="https://user-images.githubusercontent.com/94033860/166095285-d8eabe69-3cd2-47bc-8005-fb5cb6e4d03b.png">

**Comment by melohagan** [comment]
Hi @newtonlaw95 

You can access the selected rows in the **Define actions** for a button:

<img width="307" alt="Screenshot 2022-05-02 at 21 47 40" src="https://user-images.githubusercontent.com/101575380/166325113-51ab61cf-5a7d-4047-aafa-9e93e36cad6c.png">

But at the moment there is no bulk 'Save Row' action. Looping will be available in the future for actions, similar to the upcoming automations change: https://github.com/Budibase/budibase/issues/4427

You could try using a <a href="https://docs.budibase.com/docs/repeater">Repeater</a> with conditional containers, somewhat similar to this: https://github.com/Budibase/budibase/discussions/5654#discussioncomment-2662289

Instead of striped rows, you could have a button for each row that updates state (https://docs.budibase.com/docs/app-state) for that particular **_id** to reveals a dropdown and a save button for that row.

**Comment by melohagan** [comment]
You can do this now by passing through the Selected rows, and form data to an automation and looping over the IDs.

- https://docs.budibase.com/docs/looping
- https://docs.budibase.com/docs/app-action
